docs/application/tizen-studio/rt-ide/overview.md

```diff
@@ -1,8 +1,7 @@
 # TizenRT
 
 > [!NOTE]
-> RT IDE will be deprecated from Tizen Studio 4.1.
-
+> Tizen RT-IDE has been deprecated since Tizen Studio 4.1 release, but you can still browse the obsolete RT-IDE documentation using the [link](/application/tizen-studio/rt-ide/get-started/overview.md).  
```

Comment:
You had intially thought to put this at the end of the document I thought?

Reply:
@annie-samsung 

Yes, which is place is suitable as per your opinion. ? I will fix this as per your opinion.

Reply:
I would recommend to add this at the end of the document. @mijin85cho what is your thoughts?